arcsi/api/item.py

```diff
@@ -107,7 +115,7 @@ def list_items_latest():
 
 @arcsi.route("/item/<int:id>", methods=["GET"])
 @auth_token_required
-def view_item(id):
+def archon_view_item(id):
```

Comment:
Yeah, looks to be a good idea to distinguish the archon vs frontend usage like this.

Reply:
Thanks for the comments!
I double checked the API usages in the app and frontend repo, added the `/archon` prefixes into the routes where it is possible (only the Arcsi uses them itself) and marked where the legacy/old APIs are still used by the app or frontend